poetry.lock

```diff
@@ -210,7 +210,7 @@ optional = false
 python-versions = ">=3.5.0"
 
 [package.extras]
-unicode-backport = ["unicodedata2"]
+unicode_backport = ["unicodedata2"]
```

Comment:
Whats the reason for the change from "-" to "_"?

Reply:
Hmm, check the `poetry` version. I think this may be a change in 1.2.2?

Reply:
Yeah I see this is where they canonicalized extra names. https://github.com/python-poetry/poetry/pull/6541/

Reply:
🤦